cmd/syft/internal/test/integration/test-fixtures/npm-lock/node_modules/collapse-white-space/package.json

```diff
@@ -88,7 +88,7 @@
   },
   "scripts": {
     "build": "tsc",
-    "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
+    "format": "remark  -qfo && prettier  -w --loglevel warn && xo --fix",
```

Comment:
Was this a boyscout fix or removed by accident? Is the default for these just .?

Reply:
weird! I have no idea what changed this, I'll change it back